forte/processors/data_augment/algorithms/text_replacement_op.py

```diff
@@ -37,11 +38,15 @@ def __init__(self, configs: Config):
         self.configs = configs
 
     @abstractmethod
-    def replace(self, input: Entry) -> str:
+    def replace(self, input: Entry) -> Tuple[bool, str]:
```

Comment:
The input type should be `Annotation` because only `Annotation` contains span and text.